specification/hybridconnectivity/HybridConnectivity.Management/PublicCloud.tsp

```diff
@@ -357,9 +357,15 @@ interface GenerateAwsTemplate {
     @doc("ConnectorId and SolutionTypes and their properties to Generate AWS CFT Template.")
     @body
     generateAwsTemplateRequest: GenerateAwsTemplateRequest,
-  ): Azure.ResourceManager.ArmResponse<{}> | ErrorResponse;
+  ): Azure.ResourceManager.ArmResponse<HybridConnectivityPostOperationResult> | ErrorResponse;
 }
 
+/**
+ * The HybridConnectivity post operation response
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/no-empty-model"
+model HybridConnectivityPostOperationResult {}
```

Comment:
The name shoulde be `GenerateAwsTemplateReponse`, and rename to `GenerateAwsTemplateResult`.